internal/cmd/user/user.go

```diff
@@ -0,0 +1,224 @@
+package user
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/influxdata/influx-cli/v2/internal/api"
+	"github.com/influxdata/influx-cli/v2/internal/cmd"
+	"github.com/influxdata/influx-cli/v2/pkg/influxid"
+)
+
+type Client struct {
+	cmd.CLI
+	api.UsersApi
+	api.OrganizationsApi
+}
+
+type CreateParams struct {
+	cmd.OrgParams
+	Name     string
+	Password string
+}
+
+var (
+	ErrMustSpecifyOrg  = errors.New("must specify org ID or org name")
```

Comment:
Looks like there's an identical error defined here as well: https://github.com/influxdata/influx-cli/blob/e4af676ae57e51b2cea7b47498d877630de8df76/internal/cmd/delete/delete.go#L26

I feel like I've seen the org and user errors used elsewhere too maybe. Would it be worth moving them somewhere that they could be defined once and shared? Kind of like with the `ErrPasswordIsTooShort`.

Reply:
Good point, I'll do some consolidation